src/utils/validators/validateCustomDateRange.ts

```diff
@@ -0,0 +1,27 @@
+import { CourtDateRange } from "types/CaseListQueryParams"
+import { validateQueryParams } from "./validateQueryParams"
+
+export const validateCustomDateRange = (customDate: {
+  from: string | string[] | undefined
+  to: string | string[] | undefined
+}): CourtDateRange | undefined => {
+  let hasInvalidParam = false
+  Object.values(customDate).forEach((param) => {
```

Comment:
I may be wrong, but I don't think this loop is necessary, if the date parse fails on line 19-20 we will return undefined anyway

Reply:
Removed the loop and tests are still happy :)